docs/configuration/metrics/azure-storage-queue.md

```diff
@@ -13,6 +13,9 @@ The following fields need to be provided:
 
 Supported metrics:
 - MessageCount
+- Duration
```

Comment:
I'm not sure if `Duration` is a good name for this metric because it's a bit vague *what duration* it represents. How about `LongestMessageLifetime` or do you have any other suggestions?

Reply:
Kudos to @SeanFeldman 	for the good name!